iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/PropertiesXml.java

```diff
@@ -24,18 +27,31 @@
     public static void write(Identifiable<?> identifiable, NetworkXmlWriterContext context) throws XMLStreamException {
         if (identifiable.hasProperty()) {
             for (String name : identifiable.getPropertyNames()) {
-                String value = identifiable.getProperty(name);
-                context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
-                context.getWriter().writeAttribute(NAME, name);
-                context.getWriter().writeAttribute(VALUE, value);
+                if (!name.equals(Substation.GEOGRAPHICAL_TAGS_KEY)) {
+                    String value = identifiable.getProperty(name);
+                    context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
+                    context.getWriter().writeAttribute(NAME, name);
+                    context.getWriter().writeAttribute(VALUE, value);
+                } else {
+                    IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_3, context, () -> {
+                        String value = context.getAnonymizer().anonymizeString(identifiable.getProperty(name));
+                        try {
+                            context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
+                            context.getWriter().writeAttribute(NAME, name);
+                            context.getWriter().writeAttribute(VALUE, value);
+                        } catch (XMLStreamException e) {
+                            throw new UncheckedXmlStreamException(e);
+                        }
+                    });
+                }
             }
         }
     }
 
     public static void read(Identifiable identifiable, NetworkXmlReaderContext context) {
         assert context.getReader().getLocalName().equals(PROPERTY);
         String name = context.getReader().getAttributeValue(null, NAME);
-        String value = context.getReader().getAttributeValue(null, VALUE);
+        String value = context.getAnonymizer().deanonymizeString(context.getReader().getAttributeValue(null, VALUE));
```

Comment:
You should not do that: every property does not require to be anonymized/deanonymized. I would rather make geographical tags properties not anonymized at all.